youths/tests/test_youth_profiles_graphql_api.py

```diff
@@ -699,6 +699,50 @@ def test_user_cannot_update_youth_profile_with_photo_usage_field_if_under_15_yea
     )
 
 
+def test_staff_user_can_update_youth_profile_with_photo_usage_field_if_under_15_years_old(
+    rf, user_gql_client
+):
+    service = ServiceFactory(service_type=ServiceType.YOUTH_MEMBERSHIP)
+    group = GroupFactory()
+    user = user_gql_client.user
+    user.groups.add(group)
+    assign_perm("can_manage_profiles", group, service)
+    request = rf.post("/graphql")
+    request.user = user
+
+    # Under 15 years
+    profile = ProfileFactory()
```

Comment:
This will fail after base branch update and should be `profile = ProfileWithPrimaryEmailFactory()` 🕵️‍♂️